packages/ui/src/components/App.tsx

```diff
@@ -477,27 +477,42 @@ const Alerts = observer((props: AppProps) => {
   );
 });
 
-const Footer = () => (
-  <section>
-    <footer className="sticky-footer">
-      <div className="container">
-        <div className="text-center">
-          <small>
-            Get in touch on{' '}
-            <a href="https://discordapp.com/invite/Q38s3Vh">Discord</a> or {}
-            <a
-              href="https://t.me/casperlabs"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Telegram
-            </a>
-          </small>
+const Footer = () => {
+  const [latestSHA, setLatestSHA] = React.useState(null);
+  React.useEffect(() => {
+    async function myAsyncFunction() {
+      console.log('footer cmd');
+      const result = await fetch(
+        'https://api.github.com/repos/casperlabs/casper-node/commits/master'
+      );
+      const resultJson = await result.json();
```

Comment:
I don't think this is what we want. You should show the exact or version for the currently deployed instance. You could use a library to read from git, and set the environment variable, or there maybe a library that could inject the version before build.

Reply:
How could I get the currently deployed version?
The reason I did is once branches are merged to master, drone automatically deploy the result
So I thought the latest version on master is currently deployed

Reply:
The version is to be picked up from Git and not any branch. The git commit hash of your PR itself in this case would be the version. Once merged to master, it'll be present in the git log. This process doesn't need to interact with CI/CD or GitHub APIs. Just local git HEAD.